### PR TITLE
Enable sphinx-multiversion

### LINKED ISSUE #226: Migrate from sphinx-versioning to sphinx-multiversion
This should be fully compatible with newer sphinx versions, see quickstart:
https://holzhaus.github.io/sphinx-multiversion/master/quickstart.html

Here is an example of a documentation that is using sphinx-multiversion already:
https://github.com/mixxxdj/manual

### PR DESCRIPTION
**Pull request by marc1706**
Also backported the new styling

Fixes #226

### CODE REVIEW

File: development/conf.py [hunk 1]
```diff
@@ -33,7 +33,8 @@
     'sensio.sphinx.configurationblock',
     'sensio.sphinx.phpcode',
     'sensio.sphinx.bestpractice',
-    'sphinxcontrib.phpdomain'
+    'sphinxcontrib.phpdomain',
+    'sphinx_multiversion'
```
Comment (iMattPro) [inline development/conf.py:37]: do we need `sphinx_rtd_theme` in here as well?

**Comment by iMattPro** [comment]
Screenshots?

File: development/conf.py (hunk 1 above)
Comment (marc1706) [reply development/conf.py:37]: Don't think so as that's a theme and not an extension?

**Comment by marc1706** [comment]
Default view:
image
Expanded version picker:
image

I'd say looks the same :+1:

**Comment by marc1706** [comment]
Unfortunately, the master files are only in the subfolders and not in the root of the build folder. I think I'll have to add another index.html that just redirects to the main subfolder ...

File: development/conf.py (hunk 1 above)
Comment (iMattPro) [reply development/conf.py:37]: I only asked cuz I saw them doing it here

https://github.com/mixxxdj/manual/blob/2.2/source/conf.py#L35

File: development/conf.py [hunk 2]
```diff
@@ -159,7 +169,11 @@
 #html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-#html_sidebars = {}
+html_sidebars = {
+    '**': [
+        'versioning.html',
```
Comment (iMattPro) [inline development/conf.py:174]: Is this right? Or is it versions.html ?

Comment (iMattPro) [reply development/conf.py:174]: Oh, nvmnd, I think this is right. Seems to be working.

Comment (marc1706) [reply development/conf.py:174]: I think it's actually not needed as the rtd theme does not support sidebards. 🤷‍♂️

File: development/_static/css/phpbb.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    white-space: normal;
+}
+
+.rst-footer-buttons {
+    max-width: 800px;
+}
+
+.rst-versions.shift-up  {
+    overflow: auto;
+}
```
Comment (iMattPro) [inline development/_static/css/phpbb.css:24]: Shouldn't this CSS be from the other PR, it also seems a little different than in the other PR.

Comment (marc1706) [reply development/_static/css/phpbb.css:24]: This is the same CSS as before the table changes. I don't think it makes a lot of sense to port these to 3.2.x. Merging into 3.3.x and above, this should be resolved. The only addition is the rule for `.rst-versions.shift-up`.

**Review by iMattPro — APPROVED** [review]

**Review by iMattPro — APPROVED** [review]

**Pull request merged**